aws/apigateway/enable-access-logging/metadata.rego

```diff
@@ -0,0 +1,17 @@
+package cloud-metadata.aws.apigateway.enable-access-logging
+
+__rego_metadata__ := {
+	"id": "AWS061",
+	"apiVersion": 2,
+	"version": 1,
+	"title": "API Gateway stages for V1 and V2 should have access logging enabled",
+	"description": "Ensures that Amazon API Gateway API stages have Amazon CloudWatch Logs enabled.",
+	"custom": {
+		"severity": "MEDIUM",
+		"possible_Impact": "Logging provides vital information about access and usage.",
+		"urls": ["https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html", "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/apigatewayv2_stage#access_log_settings"],
+		"apis": ["APIGateway:getRestApis", "APIGateway:getStages"],
+		"iac": true,
```

Comment:
Is it worth us specifying the tfsec, cfsec etc. IDs here instead of true/false? This would make it easier to see where we implement the various rules across our toolsets.

It could be a map like:
```
{
  "tfsec": "aws-apigateway-enable-access-logging",
  "cfsec": "cf-aws-12345"
}
```

That way when we show the rule in AVD we can also list the rules that implement checks against the rule.

Reply:
This seems backwards to me. Shouldn't the product reference the metadata ID, and not the other way around?

```console
cfsec-aws-12345 --> avd-00045
tfsec-aws-02 -----> avd-00023
cspm-aws-033 -----> avd-00152
```

Reply:
Agreed that the product should reference the AVD ID, the rationale for this was that AVD can have a "Aquasec has the following tools that check for this issue" and under the hood we get a clearer view of the link than just a boolean showing if it exists

Reply:
@owenrumney that does make sense.